### DISCUSSION TITLE
Make box shadow for notes more subtle

### DISCUSSION THREAD

**Discussion by brandstetterm** [inovex/scrumlr.io#3981] [Proposals]
Hello,

in my opinion, the box shadow of the notes is way too strong. Therefore I'd suggest to make it more subtle.

Current shadow:
![Screenshot 2024-03-18 at 13 33 46](https://github.com/inovex/scrumlr.io/assets/36969812/79e7267e-b769-4a09-9926-41eb960e13ce)

Proposal 1 (more subtle, blue shadow):
![Screenshot 2024-03-18 at 13 34 06](https://github.com/inovex/scrumlr.io/assets/36969812/aecc0b65-3244-4fa3-b7b9-c49eb4d41451)

Proposal 2 (more sublte, black shadow):
![Screenshot 2024-03-18 at 13 33 40](https://github.com/inovex/scrumlr.io/assets/36969812/ca2a50a2-6d8c-44c2-8f90-3a9ea694e044)

What's your opinion on that?

**Comment by Dominik-Weinzierl** [comment]
In my opinion, I would prefer the first version (proposal 1) - it looks best to me.

**Reply by Resaki1** [reply]
Same for me!

**Comment by brandstetterm** [comment]
@SelinaBuff likes suggestion 2 (subtle black shadow) but also thinks suggestion 1 (subtle blue shadow) is OK.

**Comment by brandstetterm** [comment]
Has been implemented in #3982
